packages/playwright-core/src/utils/stackTrace.ts

```diff
@@ -47,6 +47,25 @@ export function captureRawStack(): RawStack {
   return stack.split('\n');
 }
 
+export function filterStackFile(file: string) {
+  if (!process.env.PWDEBUGIMPL && file.startsWith(CORE_DIR))
+    return false;
+  return true;
+}
+
+export function filteredStackTrace(rawStack: RawStack): StackFrame[] {
```

Comment:
I don't think we need this or previous methods, `captureLibraryStackTrace` should already cover them.

Reply:
i removed those due to `metadata.location`...